Bumbodium/Models/EmployeeViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Bumbodium.Data.DBModels;
+using Bumbodium.WebApp.Models.Utilities.ListValidation;
+using Microsoft.AspNetCore.Identity;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+using System.Xml.Linq;
+
+namespace Bumbodium.WebApp.Models
+{
+    public class EmployeeViewModel
+    {
+        public Employee Employee { get; set; }
```

Comment:
![image](https://user-images.githubusercontent.com/46578728/211146574-8f67d6b9-7b43-45b0-909a-25c3b82b4244.png)
Validation on the phone number is needed